packages/react-combobox/src/components/Option/useOption.tsx

```diff
@@ -73,6 +81,7 @@ export const useOption_unstable = (props: OptionProps, ref: React.Ref<HTMLElemen
     root: getNativeElementProps('div', {
       ref,
       role: 'option',
+      'aria-disabled': disabled ? 'true' : null,
```

Comment:
Should this be `undefined` instead of `null` for the false case?

Reply:
FWIW, I agree that `undefined` is more idiomatic React but when I tested it `null` gave the same result as `undefined`.